core/pom.xml

```diff
@@ -107,6 +107,11 @@
         </dependency>
 
         <dependency>
+            <groupId>org.jooq</groupId>
+            <artifactId>jool</artifactId>
```

Comment:
i did `....range(...).map(item -> Try.of(() -> something.Checked(item)))).map(tryGet -> tryGet.Get());`
sounds good for the dep thing